CHANGELOG.md

```diff
@@ -43,6 +43,14 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 This release also disables gossip mode by default for all components.
 See [this](docs/proposals/approved/201809_gossip-removal.md) for more details.
 
+:warning: This release moves Thanos docker images and artifacts to Golang 1.12. This release include change in GC's memory release which gives following effect (source: https://golang.org/doc/go1.12):
```

Comment:
`includes a change` ?

Reply:
We've been using Go 1.12 for awhile now, not in this release https://github.com/improbable-eng/thanos/pull/868

This release changes to 1.12.5, please change the text to note that and add link to https://github.com/improbable-eng/thanos/pull/1156